static/scripts/topicEdit.js

```diff
@@ -76,18 +76,19 @@ class TopicBlockWrapper extends React.Component {
 
                             <span className="input-group-btn">
                                 <i className="fa fa-arrows move-handle" />
-                                <a className="btn btn-secondary hidden-toggle"
+                                <button className="btn btn-secondary hidden-toggle"
                                    onClick={this.toggleHidden.bind(this)}
                                    data-toggle="tooltip"
```

Comment:
It would be good to have aria-expanded set on the button if it toggles the visibility of the section. See: https://www.w3.org/TR/wai-aria-practices/#disclosure

Reply:
This button toggles the visibility of this section on another page (for students e.g.). The section on this page itself remains visible so I think this would be unnecessary